lib/msal-browser/test/app/PublicClientApplication.spec.ts

```diff
@@ -87,19 +76,6 @@ describe("PublicClientApplication.ts Class Unit Tests", () => {
     describe("Redirect Flow Unit tests", () => {
 
         describe("handleRedirectCallback()", () => {
-
-            it("throws error if callback is not valid", async () => {
-                await expect(pca.handleRedirectCallback(null)).rejectedWith(BrowserConfigurationAuthErrorMessage.invalidCallbackObject.desc);
-                await expect(pca.handleRedirectCallback(null)).rejectedWith(BrowserConfigurationAuthError);
-            });
-
-            it("does nothing if no hash is detected", (done) => {
```

Comment:
This test should probably be converted to use `handleRedirectPromise()`

Reply:
instead of deleting it

Reply:
Done